frontend/src/queries/query.ts

```diff
@@ -102,8 +102,11 @@ async function executeQuery<N extends DataNode>(
     const currentTeamId = teamLogic.findMounted()?.values.currentTeamId
 
     if (!pollOnly) {
-        const refreshParam: RefreshType | undefined =
-            refresh && isAsyncQuery ? 'force_async' : isAsyncQuery ? 'async' : refresh
+        const refreshParam: RefreshType | undefined = isAsyncQuery
+            ? typeof refresh === 'boolean' && refresh
```

Comment:
I'm a little unsure what the behaviour should be here. There's no guarantee that `refresh` will be a boolean anymore. If the refresh param is not a boolean we just use `async`